backend/tests/unit/components/appConfig.spec.js

```diff
@@ -244,3 +245,58 @@ describe('getAppConfig', () => {
     });
   });
 });
+
+describe('getPermissionError', () => {
+  const tokenFixture = require('./fixtures/token.json');
+  const acronymDetailFixture = require('./fixtures/acronymDetail.json');
+  const configFormFixture = require('./fixtures/configForm.json');
```

Comment:
Alphabetical ordering would be nice here

Reply:
Fixed